pom.xml

```diff
@@ -45,6 +50,12 @@
             <artifactId>vertx-proton</artifactId>
             <version>${vertx-proton.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.vertx</groupId>
```

Comment:
what is the new vertx dep for?

Reply:
it removes [couple of warnings](https://travis-ci.org/radanalyticsio/streaming-amqp/builds/282785346#L2150) during the compilation. It has the scope = provided so it shouldn't increase the final jar